eth/backend.go

```diff
@@ -312,14 +313,10 @@ func (s *Ethereum) APIs() []rpc.API {
 	if s.lesServer != nil {
 		apis = append(apis, s.lesServer.APIs()...)
 	}
+
 	// Append any APIs exposed explicitly by the consensus engine
 	apis = append(apis, s.engine.APIs(s.BlockChain())...)
 
-	// Append any APIs exposed explicitly by the les server
-	if s.lesServer != nil {
-		apis = append(apis, s.lesServer.APIs()...)
-	}
-
```

Comment:
Why were the following few lines removed?

Reply:
I removed this part because I think it's a duplicate caused by a merge or something. The les APIs are already appended in line 311